docs/lease-metrics.md

```diff
@@ -3,4 +3,4 @@
 | Metric name| Metric type | Labels/tags                                                                                                                               | Status |
 | ---------- | ----------- |-------------------------------------------------------------------------------------------------------------------------------------------| ----------- |
 | kube_lease_owner | Gauge | `lease`=&lt;lease-name&gt; <br> `owner_kind`=&lt;onwer kind&gt; <br> `owner_name`=&lt;owner name&gt; <br> `namespace` = &lt;namespace&gt; <br> `lease_holder`=&lt;lease holder name&gt;| EXPERIMENTAL |
-| kube_lease_renew_time | Gauge | `lease`=&lt;lease-name&gt;                                                                                                              | EXPERIMENTAL |
+| kube_lease_renew_time | Gauge | `lease`=&lt;lease-name&gt;  <br> `namespace` = &lt;namespace&gt;                                                                        | EXPERIMENTAL |
```

Comment:
Is this file supposed to be generated? `make dockcheck` errored so i manually edited it

Reply:
I don't think we auto-generate it, we only validate the metrics in the table